src/us/kbase/narrativejobservice/sdkjobs/SDKLocalMethodRunner.java

```diff
@@ -522,10 +522,10 @@ public boolean isJobCanceled() {
                             return true;
                         }
                     } catch (Exception ex) {
-                        log.logNextLine("Non-critical error checking for job cancelation - " +
+                        log.logNextLine("ineffective attempt checking for job cancelation - " +
```

Comment:
how about "status check failed". "ineffective attempt" is definitely better than "Non-critical error" though

Reply:
The goal is to try to come up with a less scary way of saying this. `Failed` = scary

Reply:
gotcha, I think this is good for now